Index: oce-OCE-0.15/src/OpenGl/OpenGl_GlCore11.hxx
===================================================================
--- oce-OCE-0.15/src/OpenGl/OpenGl_GlCore11.hxx
+++ oce-OCE-0.15/src/OpenGl/OpenGl_GlCore11.hxx	2014-10-22 01:48:30.398756530 +0200
@@ -48,6 +48,9 @@
 #else
   #include <GL/gl.h>
   #include <GL/glu.h>
+  #include <stddef.h>
+  typedef ptrdiff_t GLsizeiptr;
+  typedef ptrdiff_t GLintptr;
 #endif
 
 #include <InterfaceGraphic.hxx>
