diff -up occt-ffce0d6/src/BRepTools/BRepTools_ShapeSet.cxx.omv~ occt-ffce0d6/src/BRepTools/BRepTools_ShapeSet.cxx
--- occt-ffce0d6/src/BRepTools/BRepTools_ShapeSet.cxx.omv~	2023-04-28 16:08:24.205230453 +0200
+++ occt-ffce0d6/src/BRepTools/BRepTools_ShapeSet.cxx	2023-04-28 16:08:29.216272417 +0200
@@ -47,9 +47,9 @@
 
 #ifdef MacOS
 #define strcasecmp(p,q) strcmp(p,q)
-#elseif _WIN32
+#elif _WIN32
 #define strcasecmp strcmp
-#elseif AIX
+#elif AIX
 #include <string.h>
 #endif
 
diff -up occt-ffce0d6/src/Poly/Poly_MakeLoops.cxx.omv~ occt-ffce0d6/src/Poly/Poly_MakeLoops.cxx
diff -up occt-ffce0d6/src/Standard/Standard_MMgrTBBalloc.cxx.omv~ occt-ffce0d6/src/Standard/Standard_MMgrTBBalloc.cxx
--- occt-ffce0d6/src/Standard/Standard_MMgrTBBalloc.cxx.omv~	2023-04-28 16:07:37.460839692 +0200
+++ occt-ffce0d6/src/Standard/Standard_MMgrTBBalloc.cxx	2023-04-28 16:07:43.785892492 +0200
@@ -18,7 +18,9 @@
 
 // paralleling with Intel TBB
 #ifdef HAVE_TBB
+#ifdef _WIN32
 #pragma comment (lib, "tbbmalloc.lib")
+#endif
 #include <tbb/scalable_allocator.h>
 using namespace tbb;
 #else
